config/v1.5/cni-metrics-helper.yaml

```diff
@@ -77,7 +77,7 @@ spec:
     spec:
       serviceAccountName: cni-metrics-helper
       containers:
-      - image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/cni-metrics-helper:v1.5.0
+      - image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/cni-metrics-helper:v1.5.5
```

Comment:
I understand why this is in here, but this kind of thing appearing in a 1.6 branch makes it all the more important to move all configuration-related stuff out of the main source repo and into the eks-charts repo where it can be properly decoupled. See #758

Reply:
Yes, completely true, the configs should be kept separate :)